### ISSUE TITLE
pyluxcoretools netconsole doesn't understand IPV6 addresses

### ISSUE THREAD

**Issue by emogenet** [LuxCoreRender/LuxCore#395]
My render farm is a bunch of physically distributed headless linux machines.

Most of the rendering nodes in that farm can only be reached over IPV6

Unfortunately, pyluxcoretools netconsole neither understands raw IPV6 addresses, nor performs DNS resolution, making it impossible to reach my render machines (my render machines all have a valid DNS name).

To wit (IP address changed to a well known IPV6 address):

python3 pyluxcoretools.zip netconsole /tmp/luxcore/0506.archi_LuxCore/00208.bcf  -s 200 -f 100 --nodes 127.0.0.1 2a00:1450:4007:80f::200e
[MainThread][2020-06-21 08:18:34,047] LuxCore 2.3
[MainThread][2020-06-21 08:18:34,048] Discovered new node: 127.0.0.1:18018
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "pyluxcoretools.zip/__main__.py", line 63, in <module>
  File "pyluxcoretools.zip/pyluxcoretools/pyluxcorenetconsole/cmd.py", line 180, in main
    elif line[0] == '?':
  File "pyluxcoretools.zip/pyluxcoretools/pyluxcorenetconsole/cmd.py", line 101, in Exec
    the remainder of the line as argument.
ValueError: too many values to unpack (expected 2)

Looking at the code, it seems to assume that ':' is a separator for the port.

**Comment by Dade916** [comment]
The ":" is used to separate IPv4 address from the (optional) port to use (for instance 127.0.0.1:1234).

However, syntax problem aside, IPv6 is not supported at all. It requires explicit IPv6 sockets creation while the code create IPv4: https://github.com/LuxCoreRender/LuxCore/blob/cba89ad993d398adc38af0ee542cf6b899f136ac/src/pyluxcoretools/pyluxcoretools/renderfarm/renderfarmjobsingleimage.py#L389

BTW, it does perform DNS resolution and you can use symbolic names instead of ip v4 addresses.

**Comment by emogenet** [comment]
Hi David,

Well aware of the current meaning of ':' for the port and how it clashes with IPV6 addresses, there is actually an entire RFC section on this specific issue: https://tools.ietf.org/html/rfc5952#section-6

As for DNS resolution, my bad, it does indeed resolve properly, it's just that the python socket stack does not support the address it resolves to (python doesn't seem to have a dual V4/V6 stack, it's icky).

As for V6 not being supported by the current code, that's correct, as I have discovered.

I have patched the code to support bothV4 and V6 and I've managed to run a render on 10 machines over V6.

If you guys are interested, I'll try to clean up the patch in coming weeks and submit a PR.

Cheers
